pkg/manager/manager.go

```diff
@@ -264,7 +264,7 @@ type Options struct {
 	// NewClient is the func that creates the client to be used by the manager.
 	// If not set this will create the default DelegatingClient that will
 	// use the cache for reads and the client for writes.
-	NewClient cluster.NewClientFunc
+	NewClient client.NewClientFunc
```

Comment:
The comment that says `DelegatingClient` should be updated.

Reply:
fixed, thanks!